supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,11 +2,44 @@
 [[links]]
 = Links
 
-// [[internal-links]]
-// == Internal links
+[[cross-references]]
+== Cross-references
 
-// [[external-links]]
-// == External links
+Follow these guidelines when adding cross-references within your documentation:
+
+* Include cross-references only when necessary.
+* Use hyperlinks and use them consistently.
+
+.Example AsciiDoc: Cross-reference
+----
+For more information about <topic>, see xref:<link>[<link_text>].
+----
+
+[[external-links]]
+== External links
+
+Follow these guidelines when linking externally:
+
+* Use hyperlinks and use them consistently.
```

Comment:
@bergerhoffer you commented in the original PR about this and I agree with you, it's not very helpful or clear what it's trying to say. Maybe just ' Use hyperlinks' ?

Reply:
Thanks @ndeevy! I adjusted this to say to use hyperlinks unless the URL is an example URL or is otherwise inaccessible. 

I also removed the hyperlink guidance from the cross-references section (because those would always be hyperlinks), and instead replaced it with some guidance on including info right there instead of xreffing if it's critical.

PTAL at these updates and let me know what you think! Diff is here: https://github.com/redhat-documentation/supplementary-style-guide/pull/240/commits/b6ed97fb477f1e4f089e471eb928f238c89486e4

Reply:
Cleaner, clearer, awesome. LGTM